src/doc/builtinplugins.rst

```diff
@@ -1754,6 +1754,12 @@ attributes are supported:
      - ptr
      - Pointer to a ``Filesystem::IOProxy`` that will handle the I/O, for
        example by reading from memory rather than the file system.
+   * - ``png:linear_premult``
+     - int
+     - If nonzero, will convert  or gamma-encoded values to linear color
```

Comment:
```suggestion
     - If nonzero, will convert sRGB or gamma-encoded values to linear color
```

Reply:
Oops, you're right. One spot I missed after changing the name I had started using at the beginning.

Reply:
Makes sense. But I think you missed the other suggested change?

Reply:
Quite right, thanks.